src/popover/popover.jsx

```diff
@@ -118,7 +120,7 @@ const Popover = React.createClass({
       position: 'fixed',
       top: anchor.top,
       left: anchor.left,
-      zIndex: 20,
+      zIndex: 0,
```

Comment:
3. on the theme https://github.com/callemall/material-ui/blob/master/src/styles/raw-themes/light-raw-theme.js#L13 ?